types/level-ttl/level-ttl-tests.ts

```diff
@@ -1,4 +1,4 @@
-import { LevelUp as levelup } from 'levelup';
+import levelup from 'levelup';
```

Comment:
https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require

Export: 
```ts
export = levelup
```

Import: 
```ts
import levelup = require("levelup")
```